app.json

```diff
@@ -22,12 +22,12 @@
     },
     "android": {
       "adaptiveIcon": {
-        "foregroundImage": "./assets/adaptive-icon.png",
+        "foregroundImage": "./src/assets/adaptive-icon.png",
         "backgroundColor": "#FFFFFF"
       }
     },
     "web": {
-      "favicon": "./assets/favicon.png"
+      "favicon": "../assets/favicon.png"
```

Comment:
**Question**: Should this be `./src/assets/favicon.png`?

```suggestion
      "favicon": "./src/assets/favicon.png"
```

Reply:
yes it should @makunomark